src/main/java/com/qa/hubspot/pages/DealsPage.java

```diff
@@ -0,0 +1,12 @@
+package com.qa.hubspot.pages;
+
+import com.qa.hubspot.base.BasePage;
+
+public class DealsPage extends BasePage{
+	
```

Comment:
where is the code?
where is page locators?
where is page actions?
Please complete this page class.